docs/UserGuide.adoc

```diff
@@ -57,7 +57,7 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 * In v 1.3: Introduce calander for meetings. Able to detect a duplicate based on a phone number. Able to fully authenticate linkedIn, and share posts to LinkedIn. All alias commands will be completed. Auto-Complete of commands enabled. Able to sort all persons in list by name alphabetically.
 
-* In v 1.4: Able to book meetings using the calender. Able to detect a duplicate based on a email address. Able to list out all Leads at user's request. CRM Book will alert you if a contact switches companies. At least two alias commands will be completed. Able to change theme statically. Able to bulk import a csv file.
+* In v 1.4: Able to book meetings using the calender. Able to detect a duplicate based on a email address. Able to list out all Leads at user's request. CRM Book will alert you if a contact switches companies. Able to change theme statically.
```

Comment:
Could you please add my "Able to bulk import a csv file" back? It seems you've removed the sentence.

Reply:
Sure. Sorry I did'nt realise that I had removed it.